frontend/src/pages/WaitingPage.tsx

```diff
@@ -33,15 +34,15 @@ export const WaitingPage = () => {
   }, [roomId, userNameOne, userNameTwo]);
 
   useEffect(() => {
-    connect("match");
+    connect(MATCH_NAMESPACE);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   useEffect(() => {
     if (!socket) {
       return;
     }
-    socket.on("connect", () => {
+    socket.on(MATCH_EVENTS.CONNECTION, () => {
```

Comment:
Can we call this match events connect instead of connection?  Since it is an action/verb